app/views/HowYouWillGetYourExpensesCurrentView.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    main_template: MainTemplate
+)
+
+@(href: String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("howYouWillGetYourExpenses.title")
+    ) {
+
+    @components.back_link()
+
+    @components.heading("howYouWillGetYourExpenses.heading")
+
+    <p>@messages("howYouWillGetYourExpenses.para1")</p>
+    <p>@messages("howYouWillGetYourExpenses.para2")</p>
+
+    <ul class="list list-bullet" role="list">
+        <li>@messages("howYouWillGetYourExpenses.item1")</li>
+        <li>@messages("howYouWillGetYourExpenses.item2")</li>
+        <li>@messages("howYouWillGetYourExpenses.item3")</li>
+        <li>@messages("howYouWillGetYourExpenses.item4")</li>
+
+    </ul>
+
+
+
+    @components.button_link(messages("howYouWillGetYourExpenses.button"), href)
```

Comment:
Href is here however it is passed in as an empty string in the controller. (See previous comments
)